build/depends.py

```diff
@@ -381,10 +381,11 @@ class SoundTouch(Dependence):
     SOUNDTOUCH_PATH = 'soundtouch-1.8.0'
 
     def sse_enabled(self, build):
-        optimize = int(util.get_flags(build.env, 'optimize', 1))
+        optimize = SCons.ARGUMENTS.get('optimize', 'portable')
         return (build.machine_is_64bit or
-                (build.toolchain_is_msvs and optimize > 2) or
-                (build.toolchain_is_gnu and optimize > 1))
+                (optimize == 'portable') or
+                (optimize == '9') or
```

Comment:
That's not good -- what I was saying is that it's not ok to break build scripts / etc. You can easily migrate the numbers to the new text options.